core/ledger/kvledger/txmgmt/statedb/statedb.go

```diff
@@ -21,6 +21,9 @@ import (
 type VersionedDBProvider interface {
 	// GetDBHandle returns a handle to a VersionedDB
 	GetDBHandle(id string, namespaceProvider NamespaceProvider) (VersionedDB, error)
+	// BootstrapDBFromSnapshot loads the public state and pvtdata hashes from the snapshot files previously
+	// generated by using the FullScanIterator
+	BootstrapDBFromState(id string, savepoint *version.Height, itr FullScanIterator, dbValueFormat byte) error
```

Comment:
Yes, I was debating a name between `BootstrapDBFromSnapshot` and `BootstrapDBFromState` (leaned towards latter because of the input parameter `itr`. As you also prefer `BootstrapDBFromSnapshot`, we can go with this name across components to be uniform.